aggregator/src/aggregator/aggregate_init_tests.rs

```diff
@@ -161,9 +162,8 @@ pub(super) struct AggregationJobInitTestCase<
     pub(super) clock: MockClock,
     pub(super) task: Task,
     pub(super) prepare_init_generator: PrepareInitGenerator<VERIFY_KEY_SIZE, V>,
-    pub(super) prepare_inits: Vec<PrepareInit>,
```

Comment:
I removed this field since it duplicated data from `aggregation_job_init_req`, and these could fall out of sync if the test updated one but not the other (as I discovered while implementing this PR).